database/dbConnection.js

```diff
@@ -0,0 +1,20 @@
+const pgp = require('pg-promise')();
+
+const herokuDB = {
+  host: process.env.HEROKU_HOST,
+  user: process.env.HEROKU_USER,
+  password: process.env.HEROKU_PW,
+  database: process.env.HEROKU_DB,
+  ssl: true,
+};
+
+const localDB = {
```

Comment:
think about this for a second are you sure you want this file to be like this? and are you using a heroku remote database cause if yes you dont need this const localDB